openmetadata-service/src/main/java/org/openmetadata/service/migration/versions/postgres/PostgresMigrationOneDotOne.java

```diff
@@ -0,0 +1,55 @@
+package org.openmetadata.service.migration.versions.postgres;
+
+import lombok.extern.slf4j.Slf4j;
+import org.jdbi.v3.core.Handle;
+import org.openmetadata.service.jdbi3.CollectionDAO;
+import org.openmetadata.service.jdbi3.locator.ConnectionType;
+import org.openmetadata.service.migration.MigrationFile;
+import org.openmetadata.service.migration.api.MigrationStep;
+
+@Slf4j
+@MigrationFile(name = "PostgresMigrationOneDotOne")
+public class PostgresMigrationOneDotOne implements MigrationStep {
+  private CollectionDAO collectionDAO;
+
+  private Handle handle;
+
+  @Override
+  public double getMigrationVersion() {
+    return 1.1;
```

Comment:
we should return 3 versions here 1.1.0 so that we can cover minor release migrations